examples/material-ui-nextjs-pages-router-ts/pages/_app.tsx

```diff
@@ -1,23 +1,15 @@
 import * as React from 'react';
 import Head from 'next/head';
 import { AppProps } from 'next/app';
+import { AppCacheProvider } from '@mui/material-nextjs/v14-pagesRouter';
```

Comment:
Apologies if I'm missing something, but why do we have these versioned paths (`v13-*` and `v14-*`)? It looks like `v14-` just re-exports everything from `v13-` so wondering if there's a reason we couldn't have `@mui/material-nextjs/pagesRouter` and `@mui/material-nextjs/appRouter`?

Reply:
Thanks for the question. It's the versioning flexibility for us to maintain. If Nextjs releases a new major (e.g. v15) with breaking changes but Material UI is still in v5, we still have room to create a new integration with Nextjs v15 and people can just bump the `material-nextjs` version and switch to `v15-*` APIs.

Reply:
Can not find a module or declarations for 
import { AppRouterCacheProvider } from '@mui/material-nextjs/v14-appRouter';